doc/design.txt

```diff
@@ -231,27 +231,27 @@ then min-write-size is 2, and so on.
 
 The fields are defined as follows:
 
-1. MAGIC: The following 16 bytes, written in host-byte-order:
-
-    const uint32_t boot_img_magic[4] = {
-        0xf395c277,
-        0x7fefd260,
-        0x0f505235,
-        0x8079b62c,
-    };
-
-2. Swap status: A series of single-byte records.  Each record corresponds to a
+1. Swap status: A series of single-byte records.  Each record corresponds to a
 flash sector in an image slot.  A swap status byte indicate the location of
 the corresponding sector data.  During an image swap, image data is moved one
 sector at a time.  The swap status is necessary for resuming a swap operation
 if the device rebooted before a swap operation completed.
 
-3. Copy done: A single byte indicating whether the image in this slot is
+2. Copy done: A single byte indicating whether the image in this slot is
 complete (0x01=done; 0xff=not done).
 
-4. Image OK: A single byte indicating whether the image in this slot has been
+3. Image OK: A single byte indicating whether the image in this slot has been
 confirmed as good by the user (0x01=confirmed; 0xff=not confirmed).
 
+4. MAGIC: The following 16 bytes, written in host-byte-order:
```

Comment:
Ah, I haven't seen that but it confuses me anyway. `host-byte-order`, `target-byte-order`, `network-byte-order`, etc all seem bad. I think it should be either `little-endian` or `big-endian`.